spec/features/task_management_spec.rb

```diff
@@ -31,11 +31,11 @@
 
     mock_complete_submission_endpoint!
     mock_submission_completed_with_task_endpoint!
-    mock_submission_completed_endpoint!
+    mock_submission_report_mi_completed_endpoint!
```

Comment:
Just a minor thing - the filenames for these fixtures are `submission_completed_no_business` and `submission_completed_report_mi` which actually read better IMHO. Worth renaming these methods to match?